### PR TITLE
Style tutorial like design

### PR DESCRIPTION
**Pull request by JesseEmond**
- Replace v-tour slots;
- Replace prev/next with new icon;
- Remove skip, consider "Quitter le tour" as 'stop', always shown;
- Basic arbitrary hover styling for now (0.8 opacity);
- Style the tutorial steps directly in HTML (that's what it was doing earlier already, I renamed it to `..._html` to make it clear;
- The previous change triggers warnings from the i18n lib, but the recommended alternative (https://kazupon.github.io/vue-i18n/guide/interpolation.html) does not seem really appropriate for a potentially arbitrary number of accented words. We can clean that up later maybe;
- Fix tutorial width for desktop.

---

image

image

image

### CODE REVIEW

File: src/assets/base.css [hunk 1]
```diff
@@ -45,6 +45,7 @@
   /* Sizes that change at certain breakpoints */
   --timeline-horizontal-padding: var(--sz-200);
   --modal-width: 75%;
+  --tutorial-width: calc(var(--sz-700) * 9);
```
Comment (edemartel) [inline src/assets/base.css]: Pourquoi `* 9`?

File: src/assets/base.css [hunk 2]
```diff
@@ -68,6 +69,7 @@
 
     --timeline-horizontal-padding: var(--sz-600);
     --modal-width: 50%;
+    --tutorial-width: calc(var(--sz-700) * 13);
```
Comment (edemartel) [inline src/assets/base.css]: Pourquoi `* 13`?

**Review by edemartel — APPROVED** [review]

File: src/assets/base.css (hunk 1 above)
Comment (JesseEmond) [reply src/assets/base.css]: Changé pour des valeurs fixes, merci

File: src/assets/base.css (hunk 2 above)
Comment (JesseEmond) [reply src/assets/base.css]: Changé pour des valeurs fixes, merci

**Pull request merged**